test/extensions/filters/http/admission_control/controller_test.cc

```diff
@@ -100,6 +100,34 @@ TEST_F(ThreadLocalControllerTest, VerifyMemoryUsage) {
   EXPECT_EQ(RequestData(3, 3), tlc_.requestCounts());
 }
 
+// Test for function: averageRps
```

Comment:
Terminate the comments with periods pls.

Reply:
Can you explain more? I don't quite understand what you mean.